mps_youtube/util.py

```diff
@@ -516,14 +520,25 @@ def _get_metadata_from_lastfm(artist, track) :
 
     data = json.loads(resp.read())
 
-    if 'track' != list(data.keys())[0] :
+    if 'track' != list(data.keys())[0]:
         return None
-    try :
+    try:
         metadata['track_title'] = data['track']['name']
         metadata['artist'] = data['track']['artist']['name']
         metadata['album'] = data['track']['album']['title']
         metadata['album_art_url'] = data['track']['album']['image'][-1]['#text']
-    except :
+    except:
         return None
 
     return metadata
+
+
+def assign_player(player):
+    try:
+        module = import_module('.{0}'.format(player), 'mps_youtube.players')
```

Comment:
This will fail to detect mpv if `PLAYER` is set to `mpv.exe`, `mpv.com`, or `/path/to/mpv`, right? These work now and should continue working.

I'm not sure how best to handle this... perhaps it's better to just require manually import players in `players/__init__.py`. #316 could be revived; the actual plugin loading code there was working, there just weren't sane APIs that could be made public (the point of #423). This `Player` class API should be reasonable for use in plugins.

Reply:
> These work now and should continue working.

Will be fixed in the next commit.

> I'm not sure how best to handle this... perhaps it's better to just require manually importing 

I really want dynamic importing to be a part of mpsyt. This way we could have an option `--install` so that any author can write a mpsyt player class and could install it with `mpsyt --install ~newPlayer.py`. This way, if we have more players in the future, their maintenance of them could be handled by their original authors.